examples/JMX/docker-compose.yaml

```diff
@@ -0,0 +1,63 @@
+  zookeeper:
+    image: confluentinc/cp-zookeeper:latest
+    ports:
+      - "2181:2181"
+      - "52181:52181"
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+      KAFKA_REST_JMX_HOSTNAME: "0.0.0.0"
+      KAFKA_REST_JMX_PORT: "52181"
+      KAFKA_JMX_OPTS: "-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.rmi.port=52181 -Dcom.sun.management.jmxremote.port=52181 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.local.only=false -Djava.rmi.server.hostname=0.0.0.0"      
+  kafka:
+    image: confluentinc/cp-kafka:latest
+    depends_on:
+      - zookeeper
+    ports:
+      - "9092:9092"
+      - "59092:59092"
+    environment:
+      KAFKA_ZOOKEEPER_CONNECT: "zookeeper:2181"
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:9092
+      KAFKA_DELETE_TOPIC_ENABLE: "true"
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR: 1
+      KAFKA_REST_JMX_HOSTNAME: "0.0.0.0"
+      KAFKA_REST_JMX_PORT: "59092"
+      KAFKA_JMX_OPTS: "-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.rmi.port=59092 -Dcom.sun.management.jmxremote.port=59092 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.local.only=false -Djava.rmi.server.hostname=0.0.0.0"
+  schema-registry:
+    image: confluentinc/cp-schema-registry:latest
+    depends_on:
+      - zookeeper
+      - kafka
+    ports:
+      - "8081:8081"
+      - "58081:58081"
+    environment:
+      SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL: "zookeeper:2181"
+      SCHEMA_REGISTRY_HOST_NAME: "schema-registry"
+      SCHEMA_REGISTRY_JMX_HOSTNAME: "0.0.0.0"
+      SCHEMA_REGISTRY_JMX_PORT: "58081"
+      SCHEMA_REGISTRY_JMX_OPTS: "-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.rmi.port=58081 -Dcom.sun.management.jmxremote.port=58081 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.local.only=false -Djava.rmi.server.hostname=0.0.0.0"
+  rest-proxy:
+    image: confluentinc/cp-kafka-rest:latest
+    depends_on:
+      - zookeeper
+      - kafka
+      - schema-registry
+    ports:
+      - "8082:8082"
+      - "58082:58082"
+    environment:
+      KAFKA_REST_ZOOKEEPER_CONNECT: "zookeeper:2181"
+      KAFKA_REST_BOOTSTRAP_SERVERS: "PLAINTEXT://kafka:9092"
+      KAFKA_REST_SCHEMA_REGISTRY_URL: "schema-registry:8081"
+      KAFKA_REST_HOST_NAME: "rest-proxy"
+      KAFKA_REST_JMX_HOSTNAME: "0.0.0.0"
+      KAFKA_REST_JMX_PORT: "58082"
+      KAFKAREST_JMX_OPTS: "-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.rmi.port=58082 -Dcom.sun.management.jmxremote.port=58082 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.local.only=false -Djava.rmi.server.hostname=0.0.0.0"
```

Comment:
shouldn't KAFKAREST_JMX_OPTS be KAFKA_REST_JMX_OPTS

Reply:
unfortunately KAFKAREST_JMX_OPTS is used in kafka-rest repo:
https://github.com/confluentinc/kafka-rest/blob/1cb07d62871d8eb24cfeb5939bbe4fc5aa308037/bin/kafka-rest-run-class#L43-L45